docs/index.md

```diff
@@ -16,7 +16,7 @@ description: |-
 terraform {
   required_providers {
     nobl9 = {
-      source  = "nobl9/nobl9"
+      source  = "nobl9.com/nobl9/nobl9"
```

Comment:
This looks weird a bit weird imo, but I guess it's correct if it was generated by a plugin.

Reply:
please revert it to be `nob9/nobl9`.

In case of self hosted provider, the source has to be an URL but in case of the official registry it has to be `{orgname}/{providername}` (exactly like with docker and dockerhub).
Our provider is published in the official registry and therefore should be `nobl9/nobl9` https://registry.terraform.io/providers/nobl9/nobl9/0.1.6